components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -260,12 +258,12 @@ - (void)layoutSubviews {
       titleCenter.y = roundf((float)titleCenter.y);
 
     } else {
-      // Title only, center vertically.
-      // +1 for slight adjustment to font baseline for centered title labels.
-      const CGFloat centeredTitleYOffset = 1.0f;
-      titleCenter.y = contentBounds.size.height - kBottomPadding - titleSize.height +
-                      centeredTitleYOffset + ((float)titleSize.height / 2.0f);
-      titleCenter.y = roundf((float)titleCenter.y);
+      // UILabels should be aligned to pixel boundaries to avoid anti-aliasing blurriness
+      CGFloat scale = self.window.contentScaleFactor;
+      if (MDCCGFloatEqual(scale, 0)) {
```

Comment:
When would this be 0?
I'm a big fan of defensive coding, but this check is unnecessary if the system will always return a non-zero value.

Reply:
This is 0 when `self.window` is `nil`. One place where this happens in MDCTabBarViewController, which calls `layoutIfNeeded` when setting the selected View Controller.

`TabBarViewControllerExample` &rarr; `-viewDidLoad` &rarr; `-loadTabBar` &rarr; `-[MDCTabBarViewController setSelectedViewController]` &rarr; `-transitionViewsWithoutAnimationFromViewController:toViewController:` &rarr; `-[MDCItemBar layoutSubviews]`